pinterest-for-woocommerce.php

```diff
@@ -44,7 +44,7 @@
 }
 
 define( 'PINTEREST_FOR_WOOCOMMERCE_PLUGIN_FILE', __FILE__ );
-define( 'PINTEREST_FOR_WOOCOMMERCE_VERSION', '1.0.1' ); // WRCS: DEFINED_VERSION.
```

Comment:
Yes, but I don't really know if this will be 1.1.0 or 2.0.0 or 1.1.x something. This is covered during the release by WooRelease `--product_version` flag the `// WRCS: DEFINED_VERSION. ` indicates this to be auto bumped by the script when the flag is present.